group_vars/all.yml.sample

```diff
@@ -397,7 +397,7 @@ dummy:
 # https://github.com/civetweb/civetweb/blob/master/docs/UserManual.md
 
 #radosgw_frontend_port: "{{ radosgw_civetweb_port if radosgw_frontend_type == 'civetweb' else '8080' }}"
-#radosgw_frontend_options: "{{ radosgw_civetweb_options if radosgw_frontend_type == 'civetweb' }}"
+#radosgw_frontend_options: "{{ radosgw_civetweb_options if radosgw_frontend_type == 'civetweb' else '' }}"
```

Comment:
we do still want the num_threads option from radosgw_civetweb_options

Reply:
according to the documentation num_threads is specific to civetweb not beast right ?